packages/trpc/server/routers/viewer/availability/schedule/get.handler.ts

```diff
@@ -30,25 +31,22 @@ export const getHandler = async ({ ctx, input }: GetOptions) => {
       name: true,
       availability: true,
       timeZone: true,
-      eventType: {
-        select: {
-          id: true,
-          eventName: true,
-          team: {
-            select: { members: { select: { userId: true } } },
-          },
-        },
-      },
     },
```

Comment:
Remove the dependancy for eventTypes. 

This was a super weird bug - if the user wasnt in any event types for that team they would always get a 401 when hitting this